database/nakadi/execute_db_scripts_in_container.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+find /docker-entrypoint-initdb.d -type f -name "*.sql" -exec psql -U "$POSTGRES_USER" -d "$POSTGRES_DB" -a -f '{}' \;
```

Comment:
It is gonna be run by postgres container. For more info see section "How to extend this image": https://hub.docker.com/_/postgres/